sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultRoleAssignmentProperties.cs

```diff
@@ -1,11 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Azure.Core;
-
-namespace Azure.Security.KeyVault.Administration
-{
-    [CodeGenModel("RoleAssignmentProperties")]
```

Comment:
We shouldn't be deleting this file. We should replace CodeGenModel with CodeGenType.